rdr_service/services/consent/validation.py

```diff
@@ -88,6 +91,14 @@ def _update_status(self, participant_id, has_valid_file):
                 summary=participant_summary,
                 session=self._session
             )
+            # Rebuild for PDR
+            self._task.execute(
+                'rebuild_one_participant_task',
+                payload={'p_id': participant_id},
+                in_seconds=30,
+                project_id=self._project_name
```

Comment:
Are there use cases where the `self._project_name` attribute here can be set to None (e.g., if StorageResultsStrategy instance was not initialized with a specific `project_id` )?   Wondering if there are cases where the call to `execute()` here would end up overriding the GCPCloudTask `project_id` GAE_PROJECT default and lead to an exception.

Reply:
You're right, good catch.

Reply:
Should be fixed now.